fleet-examples/gitrepos/longhorn-gitrepo-multicluster.yaml

```diff
@@ -0,0 +1,15 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: longhorn
+  namespace: fleet-default
```

Comment:
The other GitRepos we have use `fleet-local`. Should we keep up the consistency?

Reply:
fleet-local gitrepos are only applied to the local cluster (i.e.- where Rancher/Fleet is running). In order to be able to deploy things into clusters that are not local (i.e.- deployed or imported from Rancher) the assets need to live in the fleet-default namespace https://fleet.rancher.io/namespaces#gitrepo-namespace